integrationtest/backend/elasticsearch/src/test/resources/custom-index-mappings/complex-conflicts.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Could you add comments where the conflicts are supposed to be?

I expected to find `searchField` in here, with a type of `object` instead of `text`, so as to create a conflict, but evidently that's not the case. I'm having a hard time finding what is supposed to conflict exactly.